pkg/scheduler/framework/parallelize/parallelism.go

```diff
@@ -53,13 +54,29 @@ func chunkSizeFor(n, parallelism int) int {
 
 // Until is a wrapper around workqueue.ParallelizeUntil to use in scheduling algorithms.
 // A given operation will be a label that is recorded in the goroutine metric.
-func (p Parallelizer) Until(ctx context.Context, pieces int, doWorkPiece workqueue.DoWorkPieceFunc, operation string) {
+func (p Parallelizer) Until(ctx context.Context, pieces int, doWorkPiece func(piece int) error, operation string) error {
```

Comment:
Should we update the definition of `DoWorkPieceFunc` instead?

Reply:
`workqueue.DoWorkPieceFunc` is defined outside the scheduler and hence cannot be changed only with our use case here, wanting to observe the errors.

Reply:
As @sanposhiho commented here, I won't update the definition in this PR.